lib/galaxy/tools/actions/upload_common.py

```diff
@@ -5,6 +5,7 @@
 import socket
 import subprocess
 import tempfile
+import hashlib
```

Comment:
This import is not in alphabetical order, it should go first.

Reply:
Thanks. Changed.